src/utilities/permissions.ts

```diff
@@ -1166,7 +1212,7 @@ const featurePermissions: FeaturePermissions = {
     canUpdate: (user, _plan, preset) => queryPermissions.UPDATE_ACTIVITY_PRESET(user, preset),
   },
   commandDictionary: {
-    canCreate: user => queryPermissions.CREATE_COMMAND_DICTIONARY(user),
+    canCreate: user => queryPermissions.CREATE_DICTIONARY(user),
```

Comment:
Is this still a command dictionary permission?

Reply:
Yeah, it just is used to generically create the other dictionary types. They all use the same permission structure.